test/Microsoft.ML.CodeGenerator.Tests/ApprovalTests/ConsoleCodeGeneratorTests.cs

```diff
@@ -20,6 +20,7 @@
 using Microsoft.ML.CodeGenerator.Utilities;
 using Microsoft.ML.Data;
 using Microsoft.ML.TestFramework;
+using Microsoft.ML.TestFramework.Attributes;
```

Comment:
Since this file isn't touched, is this using necessary?

Reply:
Nope. Removed now.